tests/scripts/actionutils.sh

```diff
@@ -167,6 +167,65 @@ function create_containers() {
 
 }
 
+# functions with remote_ prefix have following assumptions:
+# sitea: node-wrk0, node-wrk1
+# siteb: node-wrk2, node-wrk3
+function remote_simple_bootstrap_two_sites() {
+    # Bootstrap sitea
+    lxc exec node-wrk0 -- sh -c "microceph cluster bootstrap"
+    lxc exec node-wrk0 -- sh -c "microceph disk add loop,2G,3"
+    tok=$(lxc exec node-wrk0 -- sh -c "microceph cluster add node-wrk1" )
+    lxc exec node-wrk1 -- sh -c "microceph cluster join $tok"
+    sleep 10
+    # Bootstrap siteb 
+    lxc exec node-wrk2 -- sh -c "microceph cluster bootstrap"
+    lxc exec node-wrk2 -- sh -c "microceph disk add loop,2G,3"
+    tok=$(lxc exec node-wrk2 -- sh -c "microceph cluster add node-wrk3" )
+    lxc exec node-wrk3 -- sh -c "microceph cluster join $tok"
+    sleep 10
+}
+
+function remote_exchange_site_tokens() {
+    set -eux
+    sitea_token=$(lxc exec node-wrk0 -- sh -c "microceph cluster export siteb")
+    siteb_token=$(lxc exec node-wrk2 -- sh -c "microceph cluster export sitea")
+
+    # perform imports on both sites
+    lxc exec node-wrk0 -- sh -c "microceph remote import siteb $siteb_token --local-name=sitea"
+    lxc exec node-wrk2 -- sh -c "microceph remote import sitea $sitea_token --local-name=siteb"
+}
+
+function remote_perform_remote_ops_check() {
+    # assumes the remote export/import operations are already performed on both sites.
+    set -eux
+    # perform ceph ops on sitea (both nodes)
+    lxc exec node-wrk0 -- sh -c "microceph.ceph -s" # local ceph status
+    lxc exec node-wrk0 -- sh -c "microceph.ceph -s --cluster siteb --id sitea" # remote ceph status
+    lxc exec node-wrk0 -- sh -c "microceph remote list --json | grep '\"name\":\"siteb\"'"
+    lxc exec node-wrk1 -- sh -c "microceph.ceph -s" # local ceph status
+    lxc exec node-wrk1 -- sh -c "microceph.ceph -s --cluster siteb --id sitea" # remote ceph status
+    lxc exec node-wrk1 -- sh -c "microceph remote list --json | grep '\"local_name\":\"sitea\"'"
+    # perform ceph ops on siteb (both nodes)
+    lxc exec node-wrk2 -- sh -c "microceph.ceph -s" # local ceph status
+    lxc exec node-wrk2 -- sh -c "microceph.ceph -s --cluster sitea --id siteb" # remote ceph status
+    lxc exec node-wrk2 -- sh -c "microceph remote list --json | grep '\"name\":\"sitea\"'"
+    lxc exec node-wrk3 -- sh -c "microceph.ceph -s" # local ceph status
+    lxc exec node-wrk3 -- sh -c "microceph.ceph -s --cluster sitea --id siteb" # remote ceph status
+    lxc exec node-wrk3 -- sh -c "microceph remote list --json | grep '\"local_name\":\"siteb\"'"
+}
+
+function remote_remove_and_verify() {
+    # Remove the configured remote from MicroCeph
+    lxc exec node-wrk0 -- sh -c "microceph remote remove siteb"
+    # Verify list output
+    match=$((lxc exec node-wrk0 -- sh -c "microceph remote list --json | grep '\"name\":\"siteb\"'") || true )
```

Comment:
Testing nit: 

I'd suggest to run a `microceph remote list` before removing the remote, to demonstrate we actually had a remote to remove in this step

Also from the action run output it looks like `microceph remote list` in this case throws an error -- I would suggest to explicitly check for that error 

```
Error: failed to fetch remotes: failed to import MicroCeph remote: no remotes configured
```

Reply:
Ack, this has been fixed.